### PR TITLE
mosnet.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you also search for subdomains here?

File: src/chrome/content/rules/mosnet.ru.xml [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+orange.mosnet.ru ³
+privateoffice.mosnet.ru ⁴
+raven.mosnet.ru ²
+speedtest.mosnet.ru ²
+speedtestlocal.mosnet.ru ²
+
+
+² refused
+³ timed out
+⁴ self signed
```
Comment (J0WI) [inline src/chrome/content/rules/mosnet.ru.xml]: Please double check, I think some are only chain issues.

Comment (galeksandrp) [inline src/chrome/content/rules/mosnet.ru.xml]: But we [don't accept domains with chain issues](https://github.com/EFForg/https-everywhere/pull/3279#issuecomment-160349032), isn't it?

Comment (galeksandrp) [inline src/chrome/content/rules/mosnet.ru.xml]: Fixed.

**Pull request merged**